### ISSUE TITLE
Checkpoints are too old

### ISSUE THREAD

**Issue by 5thofnovemberr** [moono/lpips-tf2.x#4]
Tensorflow 2 (Keras 3.1.1) only supports .keras or .h5 files but the checkpoints for this repository seem to be for tensorflow 1.x? Running the test.py I'm seeing the following error

`File format not supported: filepath=./models/vgg/exported. Keras 3 only supports V3 `.keras` and `.weights.h5` files, or legacy V1/V2 `.h5` files.` in `lpips_tensorflow.py:22`

**Comment by jangting** [comment]
I got same error message.

**Comment by jangting** [comment]
After reinstalling tensorflow 2.4.0, the error disappear.

**Comment by prabakar2532002** [comment]
after version 3.0 loading of SavedModel format is not supported in keras so to solve this issue save the models in .weights.h5 format and load from the saved .h5 model weights.

in file models/lpips_tensorflow.py the model is loaded in below line 
https://github.com/moono/lpips-tf2.x/blob/6759ae926265507d08e8e1f16f8e3a9bc520b37c/models/lpips_tensorflow.py#L22

add model saving code below and run the script to save the model file.

```
net.save_weights("vgg.weights.h5")
lin.save_weights("lin.weights.h5")
```

after saving the model replace the model load path in your code like below

```
vgg_ckpt_fn = "vgg.weights.h5"  # path to saved vgg.weights.h5 model file
lin_ckpt_fn = "lin.weights.h5"  # path to saved lin.weights.h5 model file
learned_perceptual_metric_model(image_size, vgg_ckpt_fn, lin_ckpt_fn)
```

the weights.h5 format has weights and biases variable seperately while SavedModel format has only weights variable so change `use_bias=True` in below line https://github.com/moono/lpips-tf2.x/blob/6759ae926265507d08e8e1f16f8e3a9bc520b37c/models/lpips_tensorflow.py#L175

after all of the above LPIPS loss should work fine.
